### DISCUSSION TITLE
render image alpha

### DISCUSSION THREAD

**Discussion by Sunshine19980919** [facebookresearch/pytorch3d#886] [Q&A]
How to specify the alpha of the background of the rendered image

**Comment by bottler** [comment]
You can do this with a custom shader. Each of the built-in PyTorch3D shaders should be simple to copy to your own code and adapt.

**Comment by toyosatomimi98** [comment]
What do you want to specify? I met this problem before and I make the alpha channel into binary image, and my problem solved. :->
But I'm not sure whether is the same problem with yours.
